### ISSUE TITLE
kubent 0.7.2 still calling deprecated podsecuritypolicies API

### ISSUE THREAD

**Issue by soontobemil** [doitintl/kube-no-trouble#620]
I am using kubent version 0.7.2, and it is still making calls to the deprecated podsecuritypolicies API.

**Environment**:
- Kubernetes version: 1.28.10-gke.1075001
- `kubent` version: 0.7.2

**Logs**:
Deprecated APIs called
API User agent Total calls (last 30 days) Last called
/apis/policy/v1beta1/podsecuritypolicies kubent (0.7.2/25eb8a3757d1db39a04e94bb97a3f099fb5c9fb6) 2 July 23, 2024 at 4:54:00 PM UTC-7

Steps to Reproduce:

- Run kubent on the specified Kubernetes cluster.
- Observe the deprecated API calls in the logs.

Please advise on how to avoid these deprecated API calls or provide a patch to resolve this issue.

**Comment by dark0dave** [comment]
Yeah this is cased by the old library plus us checking for old deprecations, I am working on a solution for this
